docs/developer-docs/latest/guides/auth-request.md

```diff
@@ -132,9 +163,15 @@ const { data } = await axios.get('http://localhost:1337/api/articles');
 console.log(data);
 ```
 
-Here you should receive a **403 error** because you are not allowed, as Public user, to access to the **articles**.
+In Postman: 
 
-You should use the `JWT` in the request to say that you can access to this data as **Reader user**.
+![GET request in Postman](../assets/guides/postman-get-request/GET-request-unauth.png)
```

Comment:
Sorry @ZPain8464 I was not clear enough here. I think with one more change we can merge this PR. Thanks again! Instead of the screenshot, I was thinking we could add a code block like our API documentation and use the tabs card like the above example. The code for the request is: 
::: request Postman

`GET http://localhost:1337/api/restaurants`

:::

If you want me to take care of this instead, just let me know. I would like to include this with our 7-September docs release.

Reply:
Hey @StrapiShaun thanks for clarifying. I made the adjustment, please let me know if you need anything else!